x-pack/plugins/maps/server/maps_telemetry/maps_telemetry.ts

```diff
@@ -82,6 +94,112 @@ function getIndexPatternsWithGeoFieldCount(indexPatterns: IIndexPattern[]) {
   };
 }
 
+function getEMSLayerCount(layerLists: LayerDescriptor[][]): ILayerTypeCount[] {
+  return layerLists.map((layerList: LayerDescriptor[]) => {
+    const countsById = _(layerList).countBy((layer: LayerDescriptor) => {
+      const isEmsFile =
+        layer.sourceDescriptor !== null
+          ? layer.sourceDescriptor.type === SOURCE_TYPES.EMS_FILE
+          : false;
+      const id = (layer.sourceDescriptor as AbstractSourceDescriptor).id;
+      return isEmsFile && id ? id : 'false';
+    });
+
+    const emsCountsById = countsById.pickBy((val, key) => key !== 'false');
```

Comment:
Seems rather odd that you have to filter out `false` key after gathering counts. How about filtering out none EMS file sources first and then gathering the counts?

Reply:
this code is not added by this PR, but can make that change for symmetry with `getGeoShapeAggCount`.